protobuf/concept.proto

```diff
@@ -228,15 +228,15 @@ message Thing {
         }
         // @Iterable
```

Comment:
Do we still need `@Iterable` in all of these places?

Given that the thing in the `Res` is now `repeated`, I'd say it's now explicit enough that these things are iterable, so we can delete these comments.

Reply:
Fixed.